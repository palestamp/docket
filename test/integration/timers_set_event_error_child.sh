rm ~/.docket_timer_test
set -o verbose
docket timer new work --abstract
docket timer new work_child --parent work
docket timer new work_child2 --parent work
docket timer new work_child3 --parent work

docket timer set work event:onstart:stop work_child2
