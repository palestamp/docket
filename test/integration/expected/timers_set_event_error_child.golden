docket timer new work --abstract
docket timer new work_child --parent work
docket timer new work_child2 --parent work
docket timer new work_child3 --parent work

docket timer set work event:onstart:stop work_child2
error: Event can not affect children (work child of work_child2)
